anchor/src/client/base.ts

```
@@ -133,11 +133,11 @@ export class BaseClient {
   }
 
   getManager(): PublicKey {
-    return this.provider?.publicKey || new PublicKey(0);
```

if we cannot find manager pubkey just let it fail, otherwise the error is swallowed and it's hard to debug